include/ocpp/v16/charge_point_configuration.hpp

```diff
@@ -84,6 +84,7 @@ class ChargePointConfiguration {
     KeyValue getUseSslDefaultVerifyPathsKeyValue();
     bool getVerifyCsmsCommonName();
     KeyValue getVerifyCsmsCommonNameKeyValue();
+    bool getVerifyCsmsAllowWildcards();
```

Comment:
```suggestion
    bool getVerifyCsmsAllowWildcards();
    KeyValue getVerifyCsmsAllowWildcardsKeyValue();
```

Reply:
Please also implement the `setXXX` functions when we turn these to RW and add the respective handling here:
* https://github.com/EVerest/libocpp/blob/main/lib/ocpp/v16/charge_point_configuration.cpp#L2176
* https://github.com/EVerest/libocpp/blob/main/lib/ocpp/v16/charge_point_configuration.cpp#L2478

Reply:
Are you sure the 'set' is required? This doesn't seem to be the case based on the spec (a.k.a the csms will not request it) nor could I find any example of others being set like the 'VerifyCsmsCommonName'.

Reply:
There's an argument for both variants. If we see these variables as something purely used for debugging we should make them _Readonly_ and only provide getters.
If we want this to be set by a csms (not sure about that) we should make them _ReadWrite_ and provide getters and setters

Reply:
> There's an argument for both variants. If we see these variables as something purely used for debugging we should make them _Readonly_ and only provide getters. If we want this to be set by a csms (not sure about that) we should make them _ReadWrite_ and provide getters and setters

It doesn't seem that they should be set by the CSMS, they're internal anyway (I've looked over the spec). While not sure about the 'VerifyCsmsCommonName' the wildcard is certainly not set by the CSMS. If i'm wrong, please provide me the spec part.

Reply:
To me it looks like wildcard certificates are often used in production systems, so I would expect `VerifyCsmsAllowWildcard` to be used not only for debugging. It provides more flexibility to the CSMS when e.g. migrating to another CSMS (that uses wildcard certificates). For that reason I would make them RW, altough I see that this would drop the level of security

Reply:
> > There's an argument for both variants. If we see these variables as something purely used for debugging we should make them _Readonly_ and only provide getters. If we want this to be set by a csms (not sure about that) we should make them _ReadWrite_ and provide getters and setters
> 
> It doesn't seem that they should be set by the CSMS, they're internal anyway (I've looked over the spec). While not sure about the 'VerifyCsmsCommonName' the wildcard is certainly not set by the CSMS. If i'm wrong, please provide me the spec part.

Just because they are not in the spec it doesnt mean that we can not make internal configuration keys ReadWrite

Reply:
The configuration setting has been added.